.ci/configure_sharp.sh

```diff
@@ -1,8 +1,19 @@
 #!/bin/bash -l
 
 SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+cd ${SCRIPT_DIR}
```

Comment:
same here

Reply:
This is needed because of `TOP_DIR="$(git rev-parse --show-toplevel)"` in `settings.sh`. The scripts may be called within another git repo so TOP_DIR may refer to incorrect place.